build/conan/conanfile.py

```diff
@@ -0,0 +1,141 @@
+            download("https://sourceforge.net/projects/glew/files/glew/%s/%s/download" % (self.version, zip_name), zip_name)
+            unzip(zip_name)
+            os.unlink(zip_name)
+
+    def build(self):
+        if self.settings.os == "Windows" and self.version == "master":
+            raise ConanException("Trunk builds are not supported on Windows (cannot build directly from master git repository).")
+
+        if self.settings.compiler == "Visual Studio":
+            env = VisualStudioBuildEnvironment(self)
+            with tools.environment_append(env.vars):
+                version = min(12, int(self.settings.compiler.version.value))
+                version = 10 if version == 11 else version
+                cd_build = "cd %s\\%s\\build\\vc%s" % (self.conanfile_directory, self.source_directory, version)
+                build_command = build_sln_command(self.settings, "glew.sln")
+                vcvars = vcvars_command(self.settings)
+                self.run("%s && %s && %s" % (vcvars, cd_build, build_command.replace("x86", "Win32")))
+        else:
+            if self.settings.os == "Windows":
+                replace_in_file("%s/build/cmake/CMakeLists.txt" % self.source_directory, \
+                                "if(WIN32 AND (NOT MSVC_VERSION LESS 1600)", \
+                                "if(WIN32 AND MSVC AND (NOT MSVC_VERSION LESS 1600)")
+
+            if self.version == "master":
+                self.run("make extensions")
+
+            cmake = CMake(self)
+            cmake.configure(source_dir="%s/build/cmake" % self.source_directory, defs={"BUILD_UTILS": "OFF"})
+            cmake.build()
+
+    def package(self):
+        find_glew_dir = "%s/build/conan" % self.conanfile_directory if self.version == "master" else "."
+        self.copy("FindGLEW.cmake", ".", find_glew_dir, keep_path=False)
+        self.copy("include/*", ".", "%s" % self.source_directory, keep_path=True)
+        self.copy("%s/license*" % self.source_directory, dst="licenses",  ignore_case=True, keep_path=False)
+
+        if self.settings.os == "Windows":
+            if self.settings.compiler == "Visual Studio":
+                self.copy(pattern="*.pdb", dst="bin", keep_path=False)
+                if self.options.shared:
+                    self.copy(pattern="*32.lib", dst="lib", keep_path=False)
+                    self.copy(pattern="*32d.lib", dst="lib", keep_path=False)
+                    self.copy(pattern="*.dll", dst="bin", keep_path=False)
+                else:
+                    self.copy(pattern="*32s.lib", dst="lib", keep_path=False)
+                    self.copy(pattern="*32sd.lib", dst="lib", keep_path=False)
+            else:
+                if self.options.shared:
+                    self.copy(pattern="*32.dll.a", dst="lib", keep_path=False)
+                    self.copy(pattern="*.dll", dst="bin", keep_path=False)
+                else:
+                    self.copy(pattern="*32.a", dst="lib", keep_path=False)
```

Comment:
Ok! The issue has been resolved and I've published the corrected package on bintray. Make sure you delete your local version to be sure you're using the latest one. Thanks again for the feedback!